### PR TITLE
Add oauth params necessary for implementing password credential flow

### PR DESCRIPTION
**Pull request by joystein**

### CODE REVIEW

**Comment by joystein** [comment]
Is this project alive?

**Comment by tilgovi** [comment]
Thanks, @joystein. I haven't had time or need for this project lately, but I'm happy to keep reviewing things and would love help moving this forward.

File: pyramid_oauthlib/__init__.py [hunk 1]
```diff
@@ -11,14 +11,18 @@
 
 OAUTH_PARAMS = (
     'access_token',
+    'client',
```
Comment (tilgovi) [inline pyramid_oauthlib/__init__.py]: I don't think `client` is a known parameter in the spec. Is it in popular use by some client library?

https://tools.ietf.org/html/rfc6749#section-11.2.2

Comment (joystein) [reply pyramid_oauthlib/__init__.py]: I will check if I actually use it. I wonder if I pulled it from an example somewhere.

**Comment by joystein** [comment]
Great to hear! I might have a few more minor things to contribute.

**Comment by joystein** [comment]
@tilgovi : I tested removing the client parameter and things still worked for me, so it might have been an artifact from earlier when working out the details of the integration.

**Comment by joystein** [comment]
Btw. you say you have not had any need for this project lately. Do you use something else for oauth or is it more that you work on projects where these kind of libraries are not needed?

**Comment by tilgovi** [comment]
I'm just not actively working on any Pyramid applications right now.

**Comment by tilgovi** [comment]
Thank you so much for this PR.

**Pull request merged**